@vates/types/src/xo.mts

```diff
@@ -288,7 +288,7 @@ export type XoUser = {
   id: Branded<'user'>
   name: string
   permission: string
-  pw_hash: string
+  pw_hash?: string
```

Comment:
See this [comment](https://github.com/vatesfr/xen-orchestra/pull/8494#pullrequestreview-2753152865)